calibration_assistant.cpp

```diff
@@ -36,6 +36,7 @@
 #include "phd.h"
 #include "calibration_assistant.h"
 #include "calstep_dialog.h"
+#include <algorithm>

```

Comment:
this change was already delivered to master in #1058 -- you can remove this change from the pr's branch (you can make sure your branch has the latest changes from master by either merging or rebasing)
to merge:
```
git fetch
git merge origin/master
```